jasperreport/6.1.1.wso2v2/pom.xml

```diff
@@ -0,0 +1,190 @@
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            net.sf.jasperreports.*; version="6.1.1.wso2v2",
+                            org.w3c.tools.codec.*; version="6.1.1.wso2v2"
+                        </Export-Package>
+                        <Import-Package>
+                            !net.sf.jasperreports.*; version="[6.1.0.wso2v2,7.0.0)",
+                            !org.w3c.tools.codec.*; version="[6.1.0.wso2v2,7.0.0)",
+                            com.lowagie.text.*; version="[2.1.0,3.0.0)",
+                            com.keypoint; version="[1.0.0,2.0.0)",
+                            groovyjarjarasm.asm; version="[2.3.0,3.0.0)",
+                            groovy.lang; version="[2.3.0,3.0.0)",
+                            jxl.*; version="[0.0.0,1.0.0)",
+                            javax.xml.xpath; version="[0.0.0,1.0.0)",
+                            javax.xml.transform.*; version="[0.0.0,1.0.0)",
+                            javax.xml.soap; version="[1.0.0,2.0.0)",
+                            javax.xml.parsers; version="[0.0.0,1.0.0)",
+                            javax.xml.namespace; version="[0.0.0,1.0.0)",
+                            javax.swing.*; version="[0.0.0,1.0.0)",
+                            javax.sql.*; version="[0.0.0,1.0.0)",
+                            javax.servlet.*; version="[2.6.0,3.0.0)",
+                            javax.print.*; version="[0.0.0,1.0.0)",
+                            javax.persistence; version="[2.0.0,3.0.0)",
+                            javax.naming; version="[0.0.0,1.0.0)",
+                            javax.imageio; version="[0.0.0,1.0.0)",
+                            org.xml.sax; version="[0.0.0,1.0.0)",
+                            org.w3c.dom.traversal; version="[0.0.0,1.0.0)",
+                            org.w3c.dom; version="[0.0.0,1.0.0)",
+                            org.springframework.core.io; version="[3.2.0.wso2v1,4.0.0)",
+                            org.springframework.beans.factory.*; version="[3.2.0.wso2v1,4.0.0)",
```

Comment:
Shall we omit the wso2v1 part from the version, the version range does a string comparison so this might result in unexpected outcomes when we have for example  org.springframework.beans.factory.* version 3.2.0.wso2v10 in the runtime.